README.txt

```diff
@@ -6,82 +6,101 @@ This folder contains a simulator for an ALTERNATE BIT PROTOCOL implemented in Ca
-	sender [This folder contains the unit test of the sender]
-		main.cpp
-		makefile
-		sender_input_test_ack_In.txt
-		sender_input_test_control_In.txt
-		sender_test_output.txt
-	subnet [This folder contains the unit test of the subnet]
+doc [This folder contains the documentation]
+	alternatebitprot.pdf
+	Cadmium_Documentation_Ubuntu.pdf
+	Cadmium_Windows.pdf
+
+include [This folder contains the various header files used in the project]
+	atomics [This folder contains the atomics header files ]
+		receiverCadmium.hpp
+		senderCadmium.hpp
+		subnetCadmium.hpp
+	data_structures [This folder contains the data structures headers]
+		message.hpp	
+
+lib [This folder contains the third party library files]
+	cadmium [This folder contains the cadmium (third party) library files]
+	DESTimes [This folder contains the DESTimes (third party) library files]
+	vendor [This folder contains vendor (third party) header files]
+		iestream.hpp
+		NDTime.hpp
+	
+src [This folder contains the source files used for the simualator] 
+	data_structures [This folder contains the source files from data_structures]
+		message.cpp
+	top_model [This folder contains the source files from top_model]
 		main.cpp
-		makefile
-		subnet_input_test.txt
-		subnet_test_output.txt
-
-vendor [This folder contains 3rd party header files needed in the project
-	iestream.hpp
-	NDTime.hpp
-
-top_model [This folder contains source code for the Alternate Bit Protocol simulator
-some data for testing and some simulator outputs]	
-	main.cpp
-	makefile
-	input_abp_0.txt
-	input_abp_1.txt
-	abp_output_0.txt
-	abp_output_1.txt
+
+test [This folder contains files realted to the unit tests]
+	data [This folder contains the input and output files for the unit tests]
+		sender [This folder contains some data for sender testing and simulator outputs]
+			sender_input_test_ack_In.txt
+			sender_input_test_control_In.txt
+			sender_test_output.txt
+		receiver [This folder contains some data for receiver testing and simulator outputs] 
+			receiver_input_test.txt
+			receiver_test_output.txt
+		subnet [This folder contains some data for subnet testing and simulator outputs]
+			subnet_input_test.txt
+			subnet_test_output.txt
+
+	include [This folder will include header files related to unit tests]
+
+	src [This folder contains the source code files for the unit tests]
+		sender [This folder contains source code for the sender component]
+			main.cpp
+		receiver [This folder contains source code for the receiver component] 
+			main.cpp
+		subnet [This folder contains source code for the subnet component]
+			main.cpp
+
 	
 /*************/
 /****STEPS****/
 /*************/
 
 0 - alternatebitprotocol.doc contains the explanation of this simulator
 
-1 - Update include paths in all the makefiles in this folder and subfolders. You need to update the following lines:
-	INCLUDECADMIUM=-I ../../cadmium/include
+1 - Update include paths in the makefile of this folder. You need to update the following lines:

```

Comment:
This STEP is not required anymore because  "CADMIUM" has been already included inside the project directory and necessary changes have been made inside the "makefile".

Reply:
Agreed, we thought of the same while preparing the README file but still we went with this point considering that people might download the third party folders (Cadmium) from external sources (GitHub) and place it in a different drive (i.e., local). 

However, considering that they are in lib folder (i.e., Cadmium files) of the project repository, we have removed that step in the README file and have updated the file accordingly. Many thanks for bringing this up. May I please ask you to check now and confirm? thanks!